src/Compilers/Core/Portable/CodeGen/SwitchIntegralJumpTableEmitter.SwitchBucket.cs

```diff
@@ -60,28 +82,88 @@ internal ulong BucketSize
                 }
             }
 
-            // Relative cost of the bucket
-            // roughly proportional to the number of compares it needs in the success case.
-            internal int BucketCost
+            // if a bucket could be split into two degenerate ones
+            // specifies a label index where that split would occur
```

Comment:
When you say "where that split would occur", does it occur *before* that index, *after* that index, or both?